src/Bindings/ManualBindings.cpp

```diff
@@ -3663,6 +3678,7 @@ void ManualBindings::Bind(lua_State * tolua_S)
 		// Globals:
 		tolua_function(tolua_S, "Clamp",              tolua_Clamp);
 		tolua_function(tolua_S, "StringSplit",        tolua_StringSplit);
+		tolua_function(tolua_S, "StringSplitWithQuotes", tolua_StringSplitWithQuotes);
```

Comment:
Normally we try to keep the vertical alignment in cases like these - so add the three spaces in the lines above and below this one.